COVID19/src/screens/Camera.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {View, Text} from 'react-native';
+import QRCodeScanner from 'react-native-qrcode-scanner';
+//import {RNCamera} from 'react-native-camera';
```

Comment:
If you don't need this import, remove it

Reply:
Import coment removed!